src/eterna/AnnotationManager.ts

```diff
@@ -965,7 +943,7 @@ export default class AnnotationManager {
                 const view = this.getAnnotationView(params.pose, i, params.item);
                 if (params.item.type === AnnotationHierarchyType.ANNOTATION) {
                     view.onMovedAnnotation.connect((point: Point) => {
-                        const anchorIndex = this._annotations[params.itemIndex].positions[i].anchorIndex;
+                        const anchorIndex = this.allAnnotations[params.itemIndex].positions[i].anchorIndex;
```

Comment:
Can we use `item` instead of `this.allAnnotations[params.itemIndex]`? `allAnnotations` is relatively expensive, and they're referring to the same thing anyways right?

Reply:
Good catch